sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -171,112 +182,184 @@ Sample code snippets are provided to illustrate using long-running operations [b
 
 ```python
-from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.formrecognizer import DocumentAnalysisClient
 from azure.core.credentials import AzureKeyCredential
 
-endpoint = "https://<region>.api.cognitive.microsoft.com/"
+endpoint = "https://<my-custom-subdomain>.cognitiveservices.azure.com/"
 credential = AzureKeyCredential("<api_key>")
 
-form_recognizer_client = FormRecognizerClient(endpoint, credential)
+document_analysis_client = DocumentAnalysisClient(endpoint, credential)
 model_id = "<your custom model id>"
 
 with open("<path to your form>", "rb") as fd:
-    form = fd.read()
+    document = fd.read()
 
-poller = form_recognizer_client.begin_recognize_custom_forms(model_id=model_id, form=form)
+poller = document_analysis_client.begin_analyze_document(model_id=model_id, document=document)
 result = poller.result()
 
-for recognized_form in result:
-    print("Form type: {}".format(recognized_form.form_type))
-    print("Form type confidence: {}".format(recognized_form.form_type_confidence))
-    print("Form was analyzed using model with ID: {}".format(recognized_form.model_id))
-    for name, field in recognized_form.fields.items():
-        print("Field '{}' has label '{}' with value '{}' and a confidence score of {}".format(
-            name,
-            field.label_data.text if field.label_data else name,
-            field.value,
-            field.confidence
-        ))
+for analyzed_document in result.documents:
+    print("Document has type {}".format(analyzed_document.doc_type))
+    print("Document has document type confidence {}".format(analyzed_document.confidence))
+    print("Document was analyzed with model with ID {}".format(analyzed_document.model_id))
+    for name, field in analyzed_document.fields.items():
+        print("Field '{}' has value '{}' with confidence of {}".format(name, field.value, field.confidence))
+    
+# iterate over lines, words, and selection marks on each page of the document
+for page in result.pages:
+    print("\nLines found on page {}".format(page.page_number))
+    for line in page.lines:
+        print("...Line '{}'".format(line.content))
+    for word in page.words:
+        print(
+            "...Word '{}' has a confidence of {}".format(
+                word.content, word.confidence
+            )
+        )
+    if page.selection_marks:
+        print("\nSelection marks found on page {}".format(page.page_number))
+        for selection_mark in page.selection_marks:
+            print(
+                "...Selection mark is '{}' and has a confidence of {}".format(
+                    selection_mark.state, selection_mark.confidence
+                )
+            )
+
+# iterate over tables in document
+for i, table in enumerate(result.tables):
+    print("\nTable {} can be found on page:".format(i + 1))
+    for region in table.bounding_regions:
+        print("...{}".format(i + 1, region.page_number))
```

Comment:
Is this the right code now? I think it should be clearer that bounding region returns the page number and the corresponding bounding box?

Reply:
I updated based on Paul's recommendations. Will try running these snippets shortly to ensure accuracy